sdk/textanalytics/Azure.AI.TextAnalytics/tests/AnalyzeOperationTests.cs

```diff
@@ -0,0 +1,413 @@
+
+            Assert.IsNotNull(result);
+
+            Assert.AreEqual(1, result.Count);
+
+            var keyPhrasesListId1 = new List<string> { "CEO of SpaceX", "Elon Musk", "Tesla" };
+
+            foreach (string keyphrase in result[0].KeyPhrases)
+            {
+                Assert.IsTrue(keyPhrasesListId1.Contains(keyphrase));
+            }
+        }
+
+        [Test]
+        public async Task AnalyzeOperationBatchWithErrorTest()
+        {
+            TextAnalyticsClient client = GetClient();
+
+            var documents = new List<string>
+            {
+                "Subject is taking 100mg of ibuprofen twice daily",
+                "Can cause rapid or irregular heartbeat, delirium, panic, psychosis, and heart failure.",
+                "",
+            };
+
+            AnalyzeOperationOptions operationOptions = new AnalyzeOperationOptions()
+            {
+                KeyPhrasesTaskParameters = new KeyPhrasesTaskParameters()
+                {
+                    ModelVersion = "latest"
+                },
+                DisplayName = "AnalyzeOperationBatchWithErrorTest",
+                Skip = 1
+            };
+
+            await Task.Run(() => {
+                RequestFailedException ex = Assert.ThrowsAsync<RequestFailedException>(async () =>
+                {
+                   AnalyzeOperation operation = await client.StartAnalyzeOperationBatchAsync(documents, "en", operationOptions);
+                });
+
+                Assert.IsTrue(ex.ErrorCode.Equals("InvalidArgument"));
+                Assert.IsTrue(ex.Status.Equals(400));
+            });
+
+        }
+
+        [Test]
+        public async Task AnalyzeOperationBatchWithPHIDomain()
+        {
+            TextAnalyticsClient client = GetClient();
+
+            var documents = new List<string>
+            {
+                "A patient with medical id 12345678 whose phone number is 800-102-1100 is going under heart surgery",
+            };
+
+            AnalyzeOperationOptions operationOptions = new AnalyzeOperationOptions()
+            {
+                PiiTaskParameters = new PiiTaskParameters()
+                {
+                    ModelVersion = "latest",
+                    Domain = PiiTaskParametersDomain.Phi
+                },
+                DisplayName = "AnalyzeOperationWithPHIDomain"
+            };
+
+            AnalyzeOperation operation = await client.StartAnalyzeOperationBatchAsync(documents, "en", operationOptions);
+
+            await operation.WaitForCompletionAsync();
+
+            AnalyzeOperationResult resultCollection = operation.Value;
+
+            RecognizePiiEntitiesResultCollection result = resultCollection.PiiEntitiesResult;
+
+            Assert.IsNotNull(result);
+
+            Assert.AreEqual(1, result.Count);
+
+            var redactedText = string.Empty;
+            Assert.AreEqual(redactedText, result[0].Entities.RedactedText);
```

Comment:
should this be commented out instead of fixed for the test to pass?

Reply:
commented